go/worker/merge/committee/node.go

```diff
@@ -464,9 +464,20 @@ func (n *Node) startMergeLocked(commitments []commitment.ComputeCommitment, resu
 		defer cancel()
 
 		var ioRoots, stateRoots []hash.Hash
+		var roothashMessages []*block.RoothashMessage
 		for _, result := range results {
 			ioRoots = append(ioRoots, result.IORoot)
 			stateRoots = append(stateRoots, result.StateRoot)
+
+			// Merge roothash messages.
+			// The rule is that at most one result can have sent roothash messages.
```

Comment:
Added to [list of slashing conditions](https://github.com/oasislabs/oasis-core/issues/2078) for the transaction scheduler.

Reply:
thanks